src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -1051,7 +1051,7 @@ public void onTraversalStart() {}
     public Object onTraversalSuccess() { return null; }
 
     @Override
-    protected final Object doWork() {
+    protected Object doWork() {
```

Comment:
I'm not a fan of making this non final.  We should find a way around it by adding a mechanism to provide a different progress meter.

Reply:
Maybe just a boolean-valued method that lets you disable the progress meter would suffice here

Reply:
Reverted back to final and added a boolean valued method to disable progress meter